main.go

```diff
@@ -177,6 +185,7 @@ func main() {
 	flag.Int64Var(&partitionOffset, "partition-offset", 0, "start of the partition range (only for sequential workload)")
 
 	flag.BoolVar(&measureLatency, "measure-latency", true, "measure request latency")
+	flag.BoolVar(&validateData, "validate-data", false, "write meaningful data and validate while reading")
```

Comment:
There should be a more detailed description of this option (including information how to use it) in the readme file.

Reply:
added